test/can-render-org-files.t

```diff
@@ -2,11 +2,1931 @@ Generate the base documentation
+         jsoo/universes/57/integers/0.7.0/integers_top.cma.js
+  Toplevel: js_of_ocaml --toplevel --pretty --no-cmis +dynlink.js +toplevel.js
+              prep/universes/57/integers/0.7.0/top_worker_main_integers_top.cma.bc
+              -o jsoo/universes/57/integers/0.7.0/integers_top.cma.worker.js
+  Cma: js_of_ocaml prep/universes/150/zarith/1.12/lib/zarith/zarith_top.cma
+         --pretty --wrap-with-fun zarith_top_dot_cma -o
+         jsoo/universes/150/zarith/1.12/zarith_top.cma.js
+  Toplevel: js_of_ocaml --toplevel --pretty --no-cmis +dynlink.js +toplevel.js
+              prep/universes/150/zarith/1.12/top_worker_main_zarith_top.cma.bc
+              -o jsoo/universes/150/zarith/1.12/zarith_top.cma.worker.js
+  Toplevel: js_of_ocaml --toplevel --pretty --no-cmis +dynlink.js +toplevel.js
+              prep/universes/150/zarith/1.12/top_worker_main_zarith.cma.bc -o
+              jsoo/universes/150/zarith/1.12/zarith.cma.worker.js
+  Cma: js_of_ocaml prep/universes/150/zarith/1.12/lib/zarith/zarith.cma
+         --pretty --wrap-with-fun zarith_dot_cma -o
+         jsoo/universes/150/zarith/1.12/zarith.cma.js
+  Toplevel: js_of_ocaml --toplevel --pretty --no-cmis +dynlink.js +toplevel.js
+              prep/universes/32/ctypes/0.20.2/top_worker_main_cstubs.cma.bc -o
+              jsoo/universes/32/ctypes/0.20.2/cstubs.cma.worker.js
+  Toplevel: js_of_ocaml --toplevel --pretty --no-cmis +dynlink.js +toplevel.js
+              prep/universes/32/ctypes/0.20.2/top_worker_main_ctypes.cma.bc -o
+              jsoo/universes/32/ctypes/0.20.2/ctypes.cma.worker.js
+  Cma: js_of_ocaml prep/universes/32/ctypes/0.20.2/lib/ctypes/cstubs.cma
+         --pretty --wrap-with-fun cstubs_dot_cma -o
+         jsoo/universes/32/ctypes/0.20.2/cstubs.cma.js
+  Cma: js_of_ocaml prep/universes/32/ctypes/0.20.2/lib/ctypes/ctypes.cma
+         --pretty --wrap-with-fun ctypes_dot_cma -o
+         jsoo/universes/32/ctypes/0.20.2/ctypes.cma.js
+  Toplevel: js_of_ocaml --toplevel --pretty --no-cmis +dynlink.js +toplevel.js
+              prep/universes/32/ctypes/0.20.2/top_worker_main_ctypes-top.cma.bc
+              -o jsoo/universes/32/ctypes/0.20.2/ctypes-top.cma.worker.js
+  Cma: js_of_ocaml prep/universes/32/ctypes/0.20.2/lib/ctypes/ctypes-top.cma
+         --pretty --wrap-with-fun ctypes-top_dot_cma -o
+         jsoo/universes/32/ctypes/0.20.2/ctypes-top.cma.js
+  js_of_ocaml: option '--wrap-with-fun': must be empty or a valid JavaScript
+               identifier
+  Usage: js_of_ocaml compile [OPTION]… [JS_FILES]… PROGRAM
+  Try 'js_of_ocaml compile --help' or 'js_of_ocaml --help' for more information.
+  voodoo-jsoo: internal error, uncaught exception:
+               Failure("Error: run ['js_of_ocaml'\n     'prep/universes/32/ctypes/0.20.2/lib/ctypes/ctypes-top.cma' '--pretty'\n     '--wrap-with-fun' 'ctypes-top_dot_cma' '-o'\n     'jsoo/universes/32/ctypes/0.20.2/ctypes-top.cma.js']: exited with 124")
```

Comment:
@jonludlam do you know what is supposed to be the correct value for the `--wrap-with-fun` arg here? Should it depend on something in https://github.com/jonludlam/js_top_worker?

Reply:
looks like it needs a `sed s/-/_/g` or the like.